content/sensu-go/6.0/release-notes.md

```diff
@@ -65,6 +66,64 @@ Read the [upgrade guide][1] for information about upgrading to the latest versio
 
 ---
 
+## 6.0.0 release notes
+
+**July 23, 2020** &mdash; The latest release of Sensu Go, version 6.0.0, is now available for download.
+
+**PLACEHOLDER FOR RELEASE SYNOPSIS**
+
+See the [upgrade guide][1] to upgrade Sensu to version 6.0.0.
+
+**BREAKING CHANGES FOR SENSU 6.0:**
+
+- The database schema for entities has changed. As a result, after upgrading to Sensu 6.0, you will not be able to use your database with older versions of Sensu.
+- The web UI is now a standalone product that is no longer distributed inside the `sensu-backend` binary. See the [Sensu Go Web repository][163] for more information.
+- After initial creation, you cannot change your `sensu-agent` entity configuration by modifying the agent's configuration file.
+
+**NEW FEATURES:**
+
+- ([Commercial feature][162]) PLACEHOLDER.
+- Added supported packages for Raspbian 7, 8, 9, and 10.
+- Added binary-only distributions for FreeBSD `armv5`, `armv6`, and `armv7` and Linux `ppc64le` and `s390x`.
+- Added `sensu.FetchEvent(entity, check)` and `sensu.ListEvents()` to the JavaScript filter execution environment, which means users can now query the Sensu event store for other events within the filter namespace.
+- Sensu now logs a warning when secrets cannot be sent to an agent because mTLS is not enabled.
+
+**IMPROVEMENTS:**
+
+- ([Commercial feature][162]) Added support for the `memberOf` attribute in Active Directory (AD).
+- ([Commercial feature][162]) Added more descriptive information for errors in the federated web UI.
+- ([Commercial feature][162]) Sensu now passes users' secondary groups (i.e. groups other than the Sensu user group) to `chroot` for sysvinit services, which gives the Sensu agent and backend access to the file access writes that are granted to the secondary groups.
```

Comment:
I'm not 100% sure this falls under the "Commercial feature" umbrella: this applies to the packages we publish, but one does not need a license to make use of that change. This is a package improvement for some (older) platforms.